source/about-mongosync.txt

```diff
@@ -157,6 +157,30 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+Avoid using ``mongosync`` for disaster recovery for the following reasons:
```

Comment:
Instead of having an explicit call to action, we can instead have something like this as the first sentence:

"mongosync can be run periodically against source and destination clusters to continuously synchronize between the two. Below are considerations to keep in mind if using mongosync in this way:"

Reply:
updated!